tests/particle_interpolator_bilinear_least_squares_solkz/screen-output

```diff
@@ -1,19 +1,19 @@
 -----------------------------------------------------------------------------
 -----------------------------------------------------------------------------
 
-Loading shared library <./libparticle_interpolator_bilinear_solkz.so>
+Loading shared library <./libparticle_interpolator_bilinear_least_squares_solkz.so>
 
 Number of active cells: 1,024 (on 6 levels)
 Number of degrees of freedom: 32,196 (8,450+1,089+4,225+9,216+9,216)
 
 *** Timestep 0:  t=0 years
    Skipping temperature solve because RHS is zero.
    Rebuilding Stokes preconditioner...
-   Solving Stokes system... 46+0 iterations.
+   Solving Stokes system... 47+0 iterations.
 
    Postprocessing:
-     Writing particle output:       output-particle_interpolator_bilinear_solkz/particles/particles-00000
-     Errors u_L1, p_L1, u_L2, p_L2: 2.396693e-07, 3.813761e-04, 3.798509e-07, 1.430329e-03
+     Writing particle output:       output-particle_interpolator_bilinear_least_squares_solkz/particles/particles-00000
+     Errors u_L1, p_L1, u_L2, p_L2: 1.768545e-08, 1.580101e-04, 3.863311e-08, 2.648933e-04
```

Comment:
After rebasing against master, I checked that the SolKz material model is reached at run time:
```
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".
[New Thread 0x7fffde433700 (LWP 6411)]

Thread 1 "aspect" hit Breakpoint 2, aspect::InclusionBenchmark::SolKzCompositionalMaterial<2>::evaluate (this=<optimized out>, in=..., out=...) at /home/harsha_lv/ClionProjects/aspect/tests/particle_interpolator_bilinear_least_squares_solkz/../../benchmarks/solkz/compositional_fields/solkz_compositional_fields.h:24
24	          for (unsigned int i=0; i < in.position.size(); ++i)
(gdb) bt
#0  aspect::InclusionBenchmark::SolKzCompositionalMaterial<2>::evaluate (this=<optimized out>, in=..., out=...) at /home/harsha_lv/ClionProjects/aspect/tests/particle_interpolator_bilinear_least_squares_solkz/../../benchmarks/solkz/compositional_fields/solkz_compositional_fields.h:24
#1  0x000000000085c4e7 in aspect::AdiabaticConditions::InitialProfile<2>::initialize (this=<optimized out>) at /home/harsha_lv/ClionProjects/aspect/source/adiabatic_conditions/initial_profile.cc:137
#2  0x0000000000d9b0a5 in aspect::Simulator<2>::Simulator (this=0x7fffffffb5b0, mpi_communicator_=0x7fffed547680 <ompi_mpi_comm_world>, prm=...) at /home/harsha_lv/ClionProjects/aspect/source/simulator/core.cc:533
#3  0x0000000000990170 in main (argc=2, argv=0x7fffffffdc28) at /home/harsha_lv/ClionProjects/aspect/source/main.cc:603

```
The difference in the compositional values in the statistics file is mainly because I accidentally set the test case to utilize the cell average scheme. This patch corrects that mistake.